src/manage_sql_secinfo.c

```diff
@@ -2245,29 +2377,13 @@ update_scap_cpes (int last_scap_update)
 
   element_free (element);
 
-  assert (updated_scap_cpes ? (inserts != NULL) : (inserts == NULL));
-
-  if (updated_scap_cpes)
-    {
-      sql ("%s"
-           " ON CONFLICT (uuid) DO UPDATE"
-           " SET name = EXCLUDED.name,"
-           "     title = EXCLUDED.title,"
-           "     creation_time = EXCLUDED.creation_time,"
-           "     modification_time = EXCLUDED.modification_time,"
-           "     status = EXCLUDED.status,"
-           "     deprecated_by_id = EXCLUDED.deprecated_by_id,"
-           "     nvd_id = EXCLUDED.nvd_id",
-           inserts->str);
-      g_string_free (inserts, TRUE);
-    }
+  inserts_run (&inserts);
```

Comment:
I haven't made a comparison but wouldn't it be better to reduce the transaction size, not just the size of the statements within a single transaction?

Reply:
It's an option now that the statements size is smaller, but it's very fast now (perhaps faster than it was with a single statement) and the Postgres memory usage looks to be very small (I didn't even see it in the top list, whereas it was number 1 at 20% of memory before).

Reply:
Okay, I guess if it's still a problem you can still try to optimize it further in another PR.